src/libraries/System.Net.Sockets/tests/FunctionalTests/TcpClientTest.cs

```diff
@@ -493,6 +493,34 @@ public void Connect_Dual_Success()
             }
         }
 
+        [Theory]
+        [InlineData(false, "::ffff:127.0.0.1")]
+        [InlineData(false, "127.0.0.1")]
+        [InlineData(false, "localhost")]
+        [InlineData(true, "::1")]
+        public void CtorConnect_Success(bool useIPv6, string connectString)
+        {
+            if (!Socket.OSSupportsIPv6)
+            {
+                return;
+            }
+
+            IPAddress serverAddress = useIPv6 ? IPAddress.IPv6Loopback : IPAddress.Loopback;
```

Comment:
```suggestion
        public void CtorConnect_Success(bool serverUsesIPv6, string connectString)
        {
            if (!Socket.OSSupportsIPv6)
            {
                return;
            }

            IPAddress serverAddress = serverUsesIPv6 ? IPAddress.IPv6Loopback : IPAddress.Loopback;
```

I think this would make the purpose of the parameter more obvious.